packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -63,6 +78,17 @@ export default class VTable {
                                 width: hasTd ? rect.width : undefined,
                                 height: hasTd ? rect.height : undefined,
                             };
+
+                            if (
+                                setSelectedRange &&

```

Comment:
If the cost of setting the selection range is low, we can always set it, no need to add parameter